components/normalizer/benches/decomposing_normalizer_nfd.rs

```diff
@@ -115,10 +115,10 @@ fn normalizer_bench_data() -> [BenchDataContent; 15] {
         let nfkd = &nfkd_normalizer.normalize(raw_content);
         BenchDataContent {
             file_name: file_name.to_owned(),
-            nfc: nfc.to_owned(),
-            nfd: nfd.to_owned(),
-            nfkc: nfkc.to_owned(),
-            nfkd: nfkd.to_owned(),
+            nfc: nfc.to_string(),
```

Comment:
Should the benchmarks be paying allocation costs?

Reply:
Isn't this setup code?

Reply:
oh. never mind.